presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionRelease8Suite.scala

```diff
@@ -4,10 +4,11 @@ import dotty.tools.pc.base.BaseCompletionSuite
 
 import org.junit.Test
 import org.junit.Before
+import org.junit.Ignore
 import java.nio.file.Path
 import dotty.tools.pc.utils.JRE
 
-class CompletionRelease8Suite extends BaseCompletionSuite:
+@Ignore class CompletionRelease8Suite extends BaseCompletionSuite:
```

Comment:
Looks like this still needs to be addressed?

Reply:
It doesn't work anymore, because the -release 8 option throws an error now (on purpose). I'll add the comment explaining that. If it's clearer to just remove the test completely I can do that too

Reply:
I see. Yes, in that case we should remove the test entirely.

Perhaps extract a PR that really gets rid of JDK 8 support. There seem to be several commits about that in this PR, and they should stand on their own.